### PR TITLE
chore: Bump node version to 18.x

### PR DESCRIPTION
**Pull request by tysonkamp**
Def took the long way 'round to get to 18.

### CODE REVIEW

**Comment by zeke** [comment]
Thanks.

The GitHub Actions workflow file should also be updated to run on on Node 18.

The relevant line is here: https://github.com/zeke/semantic-pull-requests/blob/021ebc531acdbd9fa882acb9371ecf480a65d17b/.github/workflows/ci.yml#L15

We can probably do without the multiple versions and just test on `18.x`

**Comment by tysonkamp** [comment]
G2G ?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [10.x, 12.x, 18.x]
```
Comment (zeke) [inline .github/workflows/ci.yml:15]:
```suggestion
        node-version: "18.x"
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [18.x]
```
Comment (zeke) [inline .github/workflows/ci.yml:15]:
```suggestion
        node-version: [16.x]
```

File: package.json [hunk 3]
```diff
@@ -40,6 +40,6 @@
     "standard": "^12.0.1"
   },
   "engines": {
-    "node": "8"
+    "node": "18"
```
Comment (zeke) [inline package.json:43]:
```suggestion
    "node": "16"
```

**Comment by zeke** [comment]
Sorry for all the back and forth here. I misread the calendar. Node.js v18 hasn't come out yet. Falling back to v16

**Comment by zeke** [comment]
Let's try it and see how it goes.

**Comment by zeke** [comment]
This should be auto-deployed to production within a few minutes. Please give it a try and let me know how it goes. If there are issues we can revert.

**Pull request merged**